htmlhint/package.json

```diff
@@ -40,6 +40,11 @@
           "default": true,
           "description": "Control whether htmlhint is enabled for HTML files or not."
         },
+        "htmlhint.extensions": {
+          "type": "array",
+          "default": ["html", "html"],
```

Comment:
I think this should be `"default": ["html", "htm"],`

Reply:
Oops, yes. Thanks!